docs/General/Manuals/client-manual.rst

```diff
@@ -61,6 +61,33 @@ The following buttons are available on the :guilabel:`Start Screen`:
 Start New Game
 --------------
 
+Clicking :guilabel:`Start new game` will cause the client to spawn a local ``freeciv21-server`` instance in
+the background and then the following dialog will appear.
+
+.. figure:: ../../_static/images/gui-elements/start-new-game.png
+    :height: 400px
+    :align: center
+    :alt: Freeciv21 Start New Game Dialog
+    :figclass: align-center
+
+    Figure 2: Start New Game Dialog
+
+
+From upper-left to lower-right, the following user interface elements are available:
```

Comment:
*From the upper left to the lower right*, I think

Reply:
fixed all references for left, right, top, bottom and center.